src/table/TablePaginationActions.jsx

```diff
@@ -39,38 +43,42 @@ export default function TablePaginationActions(props) {
   return (
     <div className={classes.root}>
       <IconButton
-        onClick={handleFirstPageButtonClick}
-        disabled={onFirstPage}
+        onClick={!onFirstPage ? handleFirstPageButtonClick : () => {}}
```

Comment:
Nice, this should solve the error message

Reply:
Yes